template/web/src/store/createWagmiConfig.ts

```diff
@@ -16,7 +16,6 @@ export function createWagmiConfig(rpcUrl: string, projectId?: string) {
     chains: [baseSepolia],
     connectors: [
       coinbaseWallet({
-        appChainIds: [baseSepolia.id],
```

Comment:
Oh was this the type issue? I believe this is required to have Smart Wallet working.

Can you confirm that without this Smart Wallet it is still working?

Reply:
Yes it was a type error:

`Object literal may only specify known properties, and 'appChainIds' does not exist in type '{ headlessMode?: false | undefined; version?: "4" | "3" | undefined; appName?: string | undefined; appLogoUrl?: string | null | undefined; preference?: "all" | "smartWalletOnly" | "eoaOnly" | undefined; keysUrl?: string | undefined; } | { ...; }'.ts(2353)`

Reply:
`wagmi` version was bumped and now the `createConfig` function does not have the `appChainIds` property

Reply:
My hunch was that `coinbaseWallet` was grabbing the `appChainId` from [`chains` ](https://github.com/coinbase/build-onchain-apps/pull/491/files#diff-849629a05d0bdce0b88b2e70e8bea52f77cb682413156c93fd260d907b27464cR16)

Reply:
`coinbaseWallet` connecter documentation does not have `appChainIds` https://wagmi.sh/react/api/connectors/coinbaseWallet